docs/change-log.md

```diff
@@ -2,6 +2,16 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update October 20th, 2023
+
+- **New:** The Humanitec Terraform Driver is now in General Availability! This major release allows more seamless integrations and management of infrastructure as code (IaC) with Terraform. Check out the [feature announcement](https://humanitec.com/blog/the-humanitec-terraform-driver-enters-general-availability) for more details.
+
+- **Improved:** Lowered the latency on all authorization-related API calls through a new authorization management system, ensuring faster and more responsive interactions across our platform.
```

Comment:
Wasn't this announced before?

Reply:
No, it was released right after the last product update on September 15th.